collada_urdf_jsk_patch occasionally fails to build

`collada_urdf_jsk_patch` package succeeds to build first time, but fails from second time.

```bash
____________________________________________________________________________________________________________________________________________________________________________________________________________________
Errors     << collada_urdf_jsk_patch:make /home/furushchev/ros/indigo/logs/collada_urdf_jsk_patch/build.make.010.log                                                                                                
make[3]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
[collada_urdf:make] /home/furushchev/ros/indigo/build/collada_urdf_jsk_patch/build/robot_model/src/collada_urdf/src/collada_urdf.cpp:66:19: fatal error: scene.h: No such file or directory                          
[collada_urdf:make]  #include <scene.h>                                                                                                                                                                              
[collada_urdf:make]                    ^                                                                                                                                                                             
[collada_urdf:make] compilation terminated.                                                                                                                                                                          
[collada_urdf:make] /home/furushchev/ros/indigo/build/collada_urdf_jsk_patch/build/robot_model/src/collada_urdf/src/collada_to_urdf.cpp:9:24: fatal error: IOSystem.hpp: No such file or directory                   
[collada_urdf:make]  #include <IOSystem.hpp>                                                                                                                                                                         
[collada_urdf:make]                         ^                                                                                                                                                                        
[collada_urdf:make] compilation terminated.                                                                                                                                                                          
[collada_urdf:make] make[6]: *** [CMakeFiles/collada_urdf.dir/src/collada_urdf.cpp.o] Error 1                                                                                                                        
[collada_urdf:make] make[5]: *** [CMakeFiles/collada_urdf.dir/all] Error 2                                                                                                                                           
[collada_urdf:make] make[5]: *** Waiting for unfinished jobs....                                                                                                                                                     
[collada_urdf:make] make[6]: *** [CMakeFiles/collada_to_urdf.dir/src/collada_to_urdf.cpp.o] Error 1                                                                                                                  
[collada_urdf:make] make[5]: *** [CMakeFiles/collada_to_urdf.dir/all] Error 2                                                                                                                                        
[collada_urdf:make] make[4]: *** [all] Error 2                                                                                                                                                                       
make[3]: *** [urdf_to_collada] Error 1
make[2]: *** [/home/furushchev/ros/indigo/devel/.private/collada_urdf_jsk_patch/lib/collada_urdf_jsk_patch/urdf_to_collada] Error 2
make[1]: *** [CMakeFiles/urdf_to_collada.dir/all] Error 2
make: *** [all] Error 2
cd /home/furushchev/ros/indigo/build/collada_urdf_jsk_patch; catkin build --get-env collada_urdf_jsk_patch | catkin env -si  /usr/bin/make --jobserver-fds=6,7 -j; cd -
```

--- a/jsk_ros_patch/collada_urdf_jsk_patch/Makefile
+++ b/jsk_ros_patch/collada_urdf_jsk_patch/Makefile
@@ -13,7 +13,7 @@ disable_ssl:
 	git config --global http.sslVerify false
 
 urdf_to_collada: disable_ssl $(GIT_DIR) patched
-	(cd build/robot_model; catkin init -w .; PKG_CONFIG_PATH=`rospack find assimp_devel`/lib/pkgconfig:${PKG_CONFIG_PATH} catkin build -v -i collada_urdf --force-cmake --no-status)
+	(cd build/robot_model; catkin init -w .; PKG_CONFIG_PATH=`rospack find assimp_devel`/lib/pkgconfig:${PKG_CONFIG_PATH} catkin build -v -i collada_urdf --force-cmake --no-status --no-notify)
 	cp $(BUILD_BIN_DIR)/urdf_to_collada .
 	cp $(BUILD_BIN_DIR)/collada_to_urdf .
 

--- a/jsk_ros_patch/collada_urdf_jsk_patch/use_assimp_devel.patch
+++ b/jsk_ros_patch/collada_urdf_jsk_patch/use_assimp_devel.patch
@@ -7,10 +7,10 @@ index 524f2e9..6bd3707 100644
  include_directories(include)
  
 -find_package(assimp QUIET)
-+find_package(assimp_devel QUIET)
++find_package(assimp_devel QUIET NAMES ASSIMP)
  if ( NOT ASSIMP_FOUND )
 -  find_package(Assimp QUIET)
-+  find_package(Assimp_Devel QUIET)
++  find_package(Assimp_Devel QUIET NAMES ASSIMP)
    if ( NOT ASSIMP_FOUND )
      find_package(PkgConfig REQUIRED)
 -    pkg_check_modules(ASSIMP assimp)

